sphinx_math_dollar/tests/test_extension.py

```diff
@@ -23,6 +25,3 @@ def _test_sphinx_build(app, status, warning):
 
     assert not status.read()
     assert not warning.read()
-
-def test_sphinx_build():
```

Comment:
Not sure if this was required or if I am missing something. Seems to work now anyway.

Reply:
Probably was required because pytest somehow didn't work with the actual test function (maybe the old test library didn't use a proper pytest fixture?). If it works now without this then we don't need it.